app/components/layout.js

```diff
@@ -40,17 +39,18 @@ class Layout extends React.Component {
   render() {
     return (
       <div>
-        <Navbar {...this.props} connect={this.connect} handleDisconnect={this.disconnect}/>
+        <Navbar {...this.props} handleConnect={this.connect} handleDisconnect={this.disconnect}/>
         <div className="container">
-          <div className="starter-template">
-            <h1>Bootstrap starter template</h1>
-            <p className="lead">
-              Use this document as a way to quickly start any new project.
-            </p>
+          <div className="row tools-wrapper">
+            <Toolbelt {...this.props} handshake={this.handshake} handleCpuSampling={this.cpuProfiling}/>
+            <div className="row">
```

Comment:
Should we even show the toolbelt while the app isn't connected?

As per the app now, if it is disconnected and someone clicks the start profiling button, and then connects, the profiling starts post a connection establishment. I think that makes it a little complicated.